src/main/java/org/opensearch/knn/common/KNNConstants.java

```diff
@@ -38,6 +38,8 @@ public class KNNConstants {
     public static final String MODEL_INDEX_NAME = ".opensearch-knn-models";
     public static final String PLUGIN_NAME = "knn";
     public static final String MODEL_METADATA_FIELD = "knn-models";
+    public static final String FILTER_FIELD = "filter";
+    public static final String TERM_FIELD = "term";
```

Comment:
They are reserved word for query. Therefore, I thought it would be better to have them where other reserved words are defined even if they are only used in test.